Put vocabulary files in consistent locations for all locales.

chrisveilleux opened the pull request:
#### Description
The recent major refactor of this skill left an inconsistent vocabulary file organization scheme.  This PR makes the scheme consistent across locales.

#### Type of PR
If your PR fits more than one category, there is a high chance you should submit more than one PR. Please consider this carefully before opening the PR.
_Either delete those that do not apply, or add an x between the square brackets like so: `- [x]`_
- [ ] Bugfix
- [ ] Feature implementation
- [X] Refactor of code (without functional changes)
- [ ] Documentation improvements
- [ ] Test improvements

#### Testing
There are no functional changes.  Skill should work as before.

#### Documentation
Not applicable to this PR

chrisveilleux committed:
Put vocabulary files in consistent locations for all locales.

krisgesling reviewed (changes requested):
- looks like the umbrella.intent should go in `vocabulary/condition`
- `fr-fr/vocabulary/next.voc` should go in date-time?

There are 7 voc files in `en-us/vocabulary` but often 5 or 6 in other languages. Are these new files that were added, so we don't yet have the translations? Seems like "like" and "outside" are

chrisveilleux committed:
moved a few more files that were out of place

chrisveilleux commented:
I looked at all the vocabulary directories and moved a few more that were out of place.  There are more files in en-us because I added them as part of the rewrite
